src/ResourceManager/Compute.ManagedService/Commands.Compute.ManagedService/Commands.Compute.ManagedService.Netcore.csproj

```diff
@@ -0,0 +1,85 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\..\..\tools\Common.Netcore.Dependencies.targets" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RuntimeIdentifiers>win10-x64</RuntimeIdentifiers>
+    <AssemblyName>Microsoft.Azure.Commands.Compute.ManagedService</AssemblyName>
+    <RootNamespace>Microsoft.Azure.Commands.Compute.ManagedService</RootNamespace>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DefineConstants>TRACE;DEBUG;NETSTANDARD</DefineConstants>
+    <OutputPath>..\..\..\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Compute.ManagedService.Netcore\</OutputPath>
+    <DelaySign>false</DelaySign>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <OutputPath>..\..\..\Package\Release\ResourceManager\AzureResourceManager\AzureRM.Compute.ManagedService.Netcore\</OutputPath>
+    <DocumentationFile></DocumentationFile>
+    <SignAssembly>True</SignAssembly>
+    <DelaySign>True</DelaySign>
+    <AssemblyOriginatorKeyFile>MSSharedLibKey.snk</AssemblyOriginatorKeyFile>
+    <DefineConstants>TRACE;RELEASE;NETSTANDARD;SIGN</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">
+
+    <PackageReference Include="System.Collections.Specialized" Version="4.3.0" />
+    <PackageReference Include="System.Reflection" Version="4.3.0" />
+    <PackageReference Include="System.Security.SecureString" Version="4.3.0" />
+    <PackageReference Include="System.Xml.XmlDocument" Version="4.3.0" />
+    <PackageReference Include="System.Xml.XmlSerializer" Version="4.3.0" />
+    <PackageReference Include="System.Security.Permissions" Version="4.4.0" />
+    <PackageReference Include="System.ServiceModel.Primitives" Version="4.4.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="..\AzureRM.Compute.ManagedService.Netcore.psd1" Link="AzureRM.Compute.ManagedService.Netcore.psd1">
```

Comment:
It looks like this is causing a problem in the Travis Build.  What, precisely does the 'link' do?

Reply:
Also, do we need the netcore module here?  Does Hyper-V have a netcore compatible module?

Reply:
I'm not sure... but I thought you requested the corresponding NetCore files...?

BTW, this is how the Compute module is done about the links:
```
https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Compute/Commands.Compute/Commands.Compute.Netcore.csproj#L134
```

Reply:
I did not request the netcore files, I requested the documentation infrastructure that is required for all modules.

Reply:
OK. Removed, cos' I'm not sure.